client/src/Results/index.js

```diff
@@ -1,6 +1,6 @@
 import React, { Component } from "react";
-import { Table, TableBody, TableHeader, TableHeaderColumn, TableRow, TableRowColumn } from 'material-ui/Table';
-import RaisedButton from 'material-ui/RaisedButton';
+import { Table, TableBody, TableHeader, TableHeaderColumn, TableRow } from 'material-ui/Table';
+import Result from "./Result";
 
 class Results extends Component {
   constructor(props) {

```

Comment:
Remove useless constructor

Reply:
The constructor will be used when we add interactivity with the calendar